etc/baryssh-client.conf

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
```

Comment:
Dnia Sun, 06 Mar 2016 03:25:16 -0800
Krzysztof Suszyński <notifications@github.com> napisał(a):

> I made them separate with fought that one can serve masked SSH service and connect to other masked SSH service, also. What do you think?

It's just a question of whether it's worth to complicate the configuration
to serve a pretty rare usage case. I doubt anyone would use it like
that. But if you believe otherwise - we can leave it as it is.

-- 
Tomasz bla Fortuna
jid: bla(at)thera.be
pgp: 0x90746E79 @ pgp.mit.edu
www: http://bla.thera.be